platform/core/PolyPodCoreSwift/Sources/PolyPodCoreSwift/Feature.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+public enum FeatureCategoryId: String {
+    case yourData
+    case knowHow
+    case tools
+    case developer
+}
+
+public struct FeatureCategory {
+    public let id: FeatureCategoryId
+    public let name: String
+    public let features: [Feature]
+}
+
+public struct Feature {
+    public init(
+        path: URL,
+        id: String,
+        name: String,
+        author: String?,
+        description: String?,
+        primaryColor: String,
+        thumbnailColor: String,
+        thumbnail: URL?,
```

Comment:
This is a super-extra-nit, but could we use the same order, whatever that is, in the description of Features for Swift and Kotlin?
Also, `path` is here an URL, a simple `String` in Kotlin. Maybe make them more uniform?

Reply:
Good catch, the order can be consistent. Now it is out of order as previous definitions were copied as defined per platform.
As for URL, it depends, swift code uses URL to load files, while Kotlin works easily with strings. An option would be to use primitives, and convert when needed.